src/engine/strat_engine/backstore/metadata/mda.rs

```diff
@@ -367,12 +367,25 @@ impl MDARegions {
     }
 }
 
+#[derive(Debug, Default, PartialEq, PartialOrd)]
+pub struct MetaDataSize(Bytes);
+
+impl MetaDataSize {
+    pub fn new(used: Bytes) -> MetaDataSize {
+        MetaDataSize(used)
+    }
+
+    pub fn bytes(&self) -> Bytes {
+        self.0
+    }
+}
+
 #[derive(Debug)]
 pub struct MDAHeader {
     last_updated: DateTime<Utc>,
 
     /// Size of region used for pool metadata.
-    used: Bytes,
+    used: MetaDataSize,
```

Comment:
Please remove or fix the comment above ```used```. It is too ambiguous to be useful. Maybe,

/// Size of the variable length metadata last written in the corresponding MDA region.
/// If no variable length has been written, the size is 0 bytes.